app/components/editor-document-title.js

```diff
@@ -13,24 +12,16 @@ export default class EditorDocumentTitleComponent extends Component {
   }
 
   get title() {
-    if (this._title) {
-      return this._title;
-    }
-    else if (this.args.title) {
-      return this.args.title;
-    }
-    else {
-      return "";
-    }
+    return this.args.title || '';
   }
 
   @action
   setTitle(event) {
-    this._title = event.target.value;
-    if (this.args.onChange) {
-      this.args.onChange(this._title);
-    }
-    if (this._title) {
+    let title = event.target.value;
+
+    this.args.onChange?.(title);
```

Comment:
wait you can optional chain a function call :open_mouth: 
mind = blown

Reply:
Yea, it look's a bit weird at first but I highly prefer it to the alternatives now :smile:.